### ISSUE TITLE
Support model merging

### ISSUE THREAD

**Issue by psychedelicious** [invoke-ai/invoke-training#110]
Model merging was removed from the main UI with plans to move that functionality to this repo. It was removed because it wasn't a super popular feature and maintaining the routes and fairly complex form UI didn't make sense given the low demand.

The merging logic itself is pretty straightforward: https://github.com/invoke-ai/InvokeAI/blob/main/invokeai/backend/model_manager/merge.py

I imagine/hope it will be pretty straightforward to add a simple Gradio UI for it.

Related issue from main UI repo: https://github.com/invoke-ai/InvokeAI/issues/6241

@RyanJDick Please correct me if I am wrong, but I think we would be happy to review a PR that adds this functionality to the training app.

**Comment by RyanJDick** [comment]
> @RyanJDick Please correct me if I am wrong, but I think we would be happy to review a PR that adds this functionality to the training app.

@psychedelicious Absolutely. I think invoke-training is the right place for that to live.
